web/src/pages/Dashboard/SubDashboards/Notes/AddNote.tsx

```diff
@@ -0,0 +1,103 @@
+import { Stack } from '@mui/material';
+import axios from 'axios';
+import Cookies from 'js-cookie';
+import '../../Dashboard.scss';
+import React, { useState } from 'react';
+import { SecondaryButton, PrimaryButton } from '../../../../component/Button';
+
+interface AddNoteProps {
+    func: React.MouseEventHandler<HTMLButtonElement>;
+    idMission: { id: number; title: string }[];
+    count: number;
+}
+
+export default function AddNote({ func, idMission, count }: AddNoteProps) {
+    const [title, setTitle] = useState('');
+    const [content, setContent] = useState('');
+    const [open, setOpen] = useState(false);
+
+    const takeTitlee = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setTitle(event.target.value);
+    };
+
+    const takeContent = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+        setContent(event.target.value);
+    };
+
+    const handleSubmit = async (
+        evt: React.MouseEvent<HTMLButtonElement, MouseEvent>
+    ) => {
+        await axios
+            .post(
+                `http://localhost:8080/note`,
+                {
+                    title,
+                    content,
+                    mission: idMission[count].id,
+                },
+                {
+                    headers: {
+                        Authorization: `Token ${Cookies.get('Token')}`,
```

Comment:
Is it not necessary to add `Content-type: application/json` here ? If it works i trust you, but it seems better

Reply:
it works like this but i can add it if you want